styles/style.styl

```diff
@@ -1,9 +1,12 @@
 
-form 
-    padding: 20px;
-
 .mybutton
     margin: 10px;
 
 .uk-search-default
     width: 200px;
+
+.size-13
+    font-size: 13px;
+
+.margin-20
```

Comment:
This is pretty specific name refering to the style used. Semantically this name should be a description of what the class does (which it is) but you shouldn't need to update the class name every time you decide to change something. Let's say you have 50 elements with a class of `.margin-20` in a month and you decide to make that spacing 25px instead. You'd have to change the class name to `margin-25` in 50 places. If you named the class something like `margin-medium` then you'd only have to change a single line in your code to affect 50 different components.

Reply:
You're right, fixed